README.md

```diff
@@ -5,22 +5,26 @@ standard library, compiler, and language spec.
 
 # How to contribute
 
+Issues and bug reports for _scala/scala_ are located in [scala/bug](https://github.com/scala/bug). This tracker is also where new contributors may find good first issues to work on.
+
 To contribute in this repo, please open a [pull request](https://help.github.com/articles/using-pull-requests/#fork--pull) from your fork of this repository.
 
+For coordinating bigger work items, we use the [scala/scala-dev tracker](https://github.com/scala/scala-dev/issues).
+
 We do have to ask you to sign the [Scala CLA](http://www.lightbend.com/contribute/cla/scala) before we can merge any of your work, to protect its open source nature.
 
-For more information on building and developing the core of Scala, make sure to read
-the rest of this README!
+The general workflow is as follows.
+1. Find/file an issue in scala/bug.
```

Comment:
We don't have a hard requirement that a scala/bug issue must exist before opening a PR in scala/scala. I'm not sure whether that's a nuance that's worth trying to capture here?

Reply:
Actually it's a reasonable sentiment. I added a new line. Hopefully it's good.